Sources/HaCWebsiteLib/ViewModels/LandingFeatures/BashWorkshop.swift

```diff
@@ -5,11 +5,11 @@ extension LandingFeatures {
   struct BashWorkshop {
     let facebookLink = "https://www.facebook.com/events/245447659318325"
     let descriptionLink = "/bash"
+    // 2017-10-25 18:45
+    let workshopStartDate = Date(timeIntervalSince1970: 1508953500)
```

Comment:
Could we maybe abstract this into a function which takes a year, month, day... and returns the relevant number? Does that function already exist?

Reply:
Am working on it